firebase-database/src/main/java/com/google/firebase/database/connection/PersistentConnectionImpl.java

```diff
@@ -428,22 +428,6 @@ public void listen(
             });
     outstandingGets.put(readId, outstandingGet);
 
-    if (!connected()) {
```

Comment:
Getting rid of this means once we issue a get, it will always be completed. The `Repo` decides whether to use, or discard, the result.